.github/workflows/ci.yml

```diff
@@ -225,7 +229,19 @@ jobs:
       - name: Build
         run: ./scripts/build-docker-image.sh "$ARCH"
 
+      - name: Export Docker Image
+        run: docker save -o relay-docker-image.tgz $IMG_VERSIONED
+
+      - name: Upload Docker Image to Artifact
+        uses: actions/upload-artifact@v3
+        with:
+          retention-days: 1
+          name: relay-docker-image
+          path: relay-docker-image.tgz
+
       - name: Push to ghcr.io
```

Comment:
If we are moving to use artifacts, can we get rid of this step to push to GHCR? There are no `docker pull`s left in the file.